### PR TITLE
change how we build test

### PR DESCRIPTION
**Pull request by anporumb**

### CODE REVIEW

File: build_functions/CMakeLists.txt [hunk 1]
```diff
@@ -47,6 +47,203 @@ function(target_link_libraries_with_arg_prefix arg_prefix whatIsBuilding lib)
...
+            add_subdirectory(${test_folder} ${test_folder}/${building})
+        endif()
+    else()
+        #message("test_folder is ${test_folder} NOT BUILDING")
+    endif()
+ 
+endfunction()
+
+#drop in replacement for build_c_tests :)
+function(build_test_artefacts whatIsBuilding use_gballoc solution_folder )
```
Comment (mattdurak) [inline build_functions/CMakeLists.txt:223]: >artefacts [](start = 20, length = 9)

artifacts

File: build_functions/CMakeLists.txt [hunk 2]
```diff
@@ -47,6 +47,203 @@ function(target_link_libraries_with_arg_prefix arg_prefix whatIsBuilding lib)
     endif()
 endfunction()
 
+########################################################################
```
Comment (mattdurak) [inline build_functions/CMakeLists.txt:50]: ha that's a lot of comment lines

File: build_functions/CMakeLists.txt (hunk 1 above)
Comment (anporumb) [reply build_functions/CMakeLists.txt:223]: Changed

---
In reply to: [455884232](https://github.com/Azure/azure-c-build-tools/pull/30#discussion_r455884232) [](ancestors = 455884232)

File: build_functions/CMakeLists.txt (hunk 2 above)
Comment (anporumb) [reply build_functions/CMakeLists.txt:50]: needed to have some delimiters about these drop-in replacements.

---
In reply to: [455885001](https://github.com/Azure/azure-c-build-tools/pull/30#discussion_r455885001) [](ancestors = 455885001)

**Pull request merged**